### PR TITLE
Pd 183 openshift shared volumes work with selinux

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px @adasilva70 Hey guys, please review this PR when you get a chance. Thanks!

File: content/portworx-install-with-kubernetes/on-premise/openshift/_index.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ series2: k8s-airgapped
 
 The following sections explain how to install Portworx with Kubernetes on OpenShift. There are two ways to
 install Portworx on OpenShift: using a DaemonSet, and using the Portworx Operator component. For clusters running OpenShift 4 and above, we recommend you install Portworx using Operator.
+
+Please note that, if you're running Portworx on Openshift 3.11 or earlier, shared volumes work only if `SELinux` is disabled on the node or in Docker. This issue is caused by a bug in Kubernetes. The bug is fixed in Kubernetes 1.13.4, which is shipped with OpenShift 4.1.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/on-premise/openshift/_index.md]: Why not make this a note? 

```suggestion
{{<info>}}
**NOTE:** if you're running Portworx on Openshift 3.11 or earlier, shared volumes work only if `SELinux` is disabled on the node or in Docker. This issue is caused by a bug in Kubernetes, which is fixed in Kubernetes 1.13.4 and is shipped with OpenShift 4.1.
{{</info>}}
```

Comment (andreipope) [reply content/portworx-install-with-kubernetes/on-premise/openshift/_index.md]: @nathan-px Just that I didn't want to draw too much attention to the matter. After all, it's solved in recent versions. Anyway, let's make it a note.

**Comment by andreipope** [comment]
@adasilva70 Hey Alex, can you please review this PR?

**Comment by andreipope** [comment]
@luayalem Hey Luay, can you please review this PR?

**Comment by andreipope** [comment]
> I am not sure we can say `This issue is caused by a bug in Kubernetes, which is fixed in Kubernetes 1.13.4 and is shipped with OpenShift 4.1.`. I searched in the K8s 1.13.4 and I could not find anything related to selinux. Do we know what the buy was and what PR# ?.

@luayalem It's not specified in the original JIRA (https://portworx.atlassian.net/browse/PD-183) but I did a quick google search and found this: https://bugzilla.redhat.com/show_bug.cgi?id=1634158. Can't say if it's related to Kubernetes or OpenShift though.

I think we can just say something like this:

```
This issue is fixed in OpenShift 4.1.
```

Although, according to Bugzilla, it looks like it has been fixed in an earlier version:

```
Checked on below version:
openshift v3.9.68
kubernetes v1.9.1+a0ce1bc657
```


@adasilva70 What's your take on this?

**Comment by andreipope** [comment]
> > I am not sure we can say `This issue is caused by a bug in Kubernetes, which is fixed in Kubernetes 1.13.4 and is shipped with OpenShift 4.1.`. I searched in the K8s 1.13.4 and I could not find anything related to selinux. Do we know what the buy was and what PR# ?.
> 
> @luayalem It's not specified in the JIRA ticket (https://portworx.atlassian.net/browse/PD-183) but I did a quick google search and found this: https://bugzilla.redhat.com/show_bug.cgi?id=1634158. Can't say if it's related to Kubernetes or OpenShift though.
> 
> I think we can just say something like this:
> 
> ```
> This issue is fixed in OpenShift 4.1.
> ```
> 
> Although, according to Bugzilla, it looks like it has been fixed in an earlier version:
> 
> ```
> Checked on below version:
> openshift v3.9.68
> kubernetes v1.9.1+a0ce1bc657
> ```
> 
> @adasilva70 What's your take on this?

And we can add a link to https://bugzilla.redhat.com/show_bug.cgi?id=1634158.

**Comment by adasilva70** [comment]
Yes, you can add the link to bugzilla. Most customers have a .0 version, like Openshif 3.11 will have kubernetes 1.11.0, hence they will not see the fix (which is on k8s 1.11.4 I believe)

**Pull request merged**